test/lib/usd/plugin/nullApiExporter.cpp

```diff
@@ -129,6 +118,17 @@ REGISTER_EXPORT_JOB_CONTEXT_UI_FCT(Curly)
     return forcedSettings;
 }
 
+REGISTER_EXPORT_JOB_CONTEXT_FCT(
+    Curly,
+    "Curly's special",
+    "Test coverage of error handling part deux")
+{
+    VtDictionary extraArgs;
+    // Incorrect type:
+    extraArgs[UsdMayaJobExportArgsTokens->apiSchema] = VtValue(std::string("testApi"));
+    return extraArgs;
+}
+
```

Comment:
I don't see any changes other than moving the macro. Is there a reason this macro moved below the other one?

Reply:
Ah yes perfect. Thanks for thinking of that.